app/src/main/java/com/github/studydistractor/sdp/account/CreateAccountActivity.kt

```diff
@@ -0,0 +1,213 @@
+            horizontalAlignment = Alignment.CenterHorizontally
+        ) {
+
+            Text(
+                text = if (selectedDateText.isNotEmpty()) {
+                    "Your birthday is $selectedDateText"
+                } else {
+                    "Please pick your birthday"
+                }
+            )
+
+            // Birthday button
+            Button(
+                onClick = {
+                    datePicker.show()
+                },
+                colors = ButtonDefaults.buttonColors(backgroundColor = Color.White),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = 8.dp)
+                    .testTag("birthday"),
+
+            ) {
+                Icon(Icons.Filled.CardGiftcard, contentDescription = null)
+                Text(text = "Select your birthday")
+            }
+
+            // Firstname Text Field
+            OutlinedTextField(
+                value = firstname,
+                onValueChange = { firstname = it },
+                label = { Text("Firstname") },
+                singleLine = true,
+                keyboardOptions = KeyboardOptions.Default.copy(
+                    keyboardType = KeyboardType.Email
+                ),
+                leadingIcon = {
+                    Icon(Icons.Filled.Person, contentDescription = null)
+                },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = 8.dp)
+                    .testTag("firstname")
+            )
+
+            // LastName Text Field
+            OutlinedTextField(
+                value = lastname,
+                onValueChange = { lastname = it },
+                label = { Text("Lastname") },
+                singleLine = true,
+                keyboardOptions = KeyboardOptions.Default.copy(
+                    keyboardType = KeyboardType.Email
+                ),
+                leadingIcon = {
+                    Icon(Icons.Filled.Person2, contentDescription = null)
+                },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = 8.dp)
+                    .testTag("lastname")
+            )
+
+            // Phone Text Field
+            OutlinedTextField(
+                value = phone,
+                onValueChange = { phone = it },
+                label = { Text("Phone") },
+                singleLine = true,
+                keyboardOptions = KeyboardOptions.Default.copy(
+                    keyboardType = KeyboardType.Email
+                ),
+                leadingIcon = {
+                    Icon(Icons.Filled.Phone, contentDescription = null)
+                },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(vertical = 8.dp)
+                    .testTag("phone")
+            )
+
+            // Validate
+            Button( onClick = {
```

Comment:
The way to do input validation in Jetpack Compose is like this : https://proandroiddev.com/input-validation-in-jetpack-compose-e99c18b44fe3